pkg/controller/nodeipam/config/types.go

```diff
@@ -31,4 +31,9 @@ type NodeIPAMControllerConfiguration struct {
 	// NodeCIDRMaskSizeIPv6 is the mask size for IPv6 node cidr in dual-stack cluster.
 	// This can be used only with dual stack clusters and is incompatible with single stack clusters.
 	NodeCIDRMaskSizeIPv6 int32
+	// EnableMultiSubnetCluster is bound to a command-line flag. When true, it enables
+	// generating nodeTopology custom resource based on node's subnetwork configuration,
+	// which is represented a node label. Enabling this feature also ensures that a
+	// nodeTopology CR named 'default' is already installed.
```

Comment:
(Have a similar comment within `pkg/controller/nodeipam/ipam/cloud_cidr_allocator.go` as well)

typo: Possible change: "which is represented a node label" -> "which is represented by a node label" 

Also regarding _"Enabling this feature also ensures that a nodeTopology CR named 'default' is already installed."_, want to make sure we are on the same page. Is this trying to say that it "assumes" that a nodeTopologyCR named 'default' is already installed, since we ourselves don't create the resource?

Reply:
>  Is this trying to say that it "assumes" that a nodeTopologyCR named 'default' is already installed, since we ourselves don't create the resource?

Yep. The resource was pre-installed from the pod manifest in CCM component inside gke. There is only one nodetopology CR. For OSS users, it will fail and retry forever if the CR is not installed already in their pod manifest.